packages/crypto/tests/signature-fixtures.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "algorithm": "ES256",
+    "publicKeyDid": "did:key:zDnaeSQxcWyYLQQkQm6kdofoQYZ3FTeBrVDQ7ATsH5x7jDtan",
+    "publicKeyMultibase": "zdSdJswiffwCx8wRyw9h2tEd1Qq4iiqsyDcZqX8Xor7Xg",
+    "messageBase64": "oWVoZWxsb2V3b3JsZA",
+    "signatureBase64": "4x9P4AMMUjMBazqFS+97XQeb1K2bilrbjOwqhUl8d+lykX/UQfT0U3QBePkX0JU0cx7PPrJ46UCU5bA3YNCmYQ"
```

Comment:
actually had to replace this test vector because the previous signature had a `lowS` which we no longer allow

Reply:
Oh funny!  I bet you already did this, but I double-checked that the new vectors work just fine without the changes in this PR.